### PR TITLE
Update learn HTML5 web forms page

### PR DESCRIPTION
**Pull request by estelle**
all these features have been supported for a very long time, so made the examples live and updated text to remove "when supported" and "fallback" type language.

### CODE REVIEW

File: files/en-us/learn/forms/html5_input_types/index.md [hunk 1]
```diff
@@ -29,30 +29,33 @@ In the [previous article](/en-US/docs/Learn/Forms/Basic_native_form_controls) we
...
-> Most of the features discussed in this article have wide support across browsers. We'll note any exceptions. If you want more detail on browser support, you should consult our [HTML forms element reference](/en-US/docs/Web/HTML/Element#forms), and in particular our extensive [\<input> types](/en-US/docs/Web/HTML/Element/input) reference.
-
 Because HTML form control appearance may be quite different from a designer's specifications, web developers sometimes build their own custom form controls. We cover this in an advanced tutorial: [How to build custom form widgets](/en-US/docs/Learn/Forms/How_to_build_custom_form_controls).
 
 ## Email address field
 
 This type of field is set using the value `email` for the [`type`](/en-US/docs/Web/HTML/Element/input#type) attribute:
 
-```html
+```html hidden live-sample___email
```
Comment (hamishwillee) [inline files/en-us/learn/forms/html5_input_types/index.md:38]: FMI what's this `live-sample___email` text? Not seen before.

Comment (hamishwillee) [reply files/en-us/learn/forms/html5_input_types/index.md:38]: Ah, a way of highlighting particular code as part of an example. 

Why not put both of these blocks in the same code block?

File: files/en-us/learn/forms/html5_input_types/index.md [hunk 2]
```diff
@@ -29,30 +29,33 @@ In the [previous article](/en-US/docs/Learn/Forms/Basic_native_form_controls) we
...
+```
+
+```html live-sample___email
 <input type="email" id="email" name="email" />
 ```
 
-When this [`type`](/en-US/docs/Web/HTML/Element/input#type) is used, the user is required to type a valid email address into the field. Any other content causes the browser to display an error when the form is submitted. You can see this in action in the below screenshot.
+{{EmbedLiveSample('email','100%','50')}}
+
+When this [`type`](/en-US/docs/Web/HTML/Element/input#type) is used, the value must be an email address to be valid. Any other content causes the browser to display an error when the form is submitted. You can see this in action in the below screenshot.
```
Comment (hamishwillee) [inline files/en-us/learn/forms/html5_input_types/index.md:48]:
```suggestion
When this [`type`](/en-US/docs/Web/HTML/Element/input#type) is used, the value must be an email address to be valid. Any other content causes the browser to display an error when the form is submitted. You can see this in action in the screenshot below.
```

File: files/en-us/learn/forms/html5_input_types/index.md [hunk 3]
```diff
@@ -193,97 +224,146 @@ price.addEventListener("input", () => {
...
 
 ## Date and time pickers
 
-Gathering date and time values has traditionally been a nightmare for web developers. For a good user experience, it is important to provide a calendar selection UI, enabling users to select dates without necessitating context switching to a native calendar application or potentially entering them in differing formats that are hard to parse. The last minute of the previous millennium can be expressed in the following different ways, for example: 1999/12/31, 23:59 or 12/31/99T11:59PM.
+Generally, for a good user experience when gathering date and time values, it is important to provide a calendar selection UI, enabling users to select dates without necessitating context switching to a native calendar application or potentially entering them in differing formats that are hard to parse. The last minute of the previous millennium can be expressed in the following different ways: `1999/12/31`, `23:59`, or `12/31/99T11:59PM`.
```
Comment (hamishwillee) [inline files/en-us/learn/forms/html5_input_types/index.md:241]:
```suggestion
Generally, for a good user experience when gathering date and time values, it is important to provide a calendar selection UI. These enables users to select dates without needing to context switch to a native calendar application or potentially entering them in differing formats that are hard to parse. The last minute of the previous millennium can be expressed in the following different ways: `1999/12/31`, `23:59`, or `12/31/99T11:59PM`.
```

File: files/en-us/learn/forms/html5_input_types/index.md [hunk 4]
```diff
@@ -193,97 +224,146 @@ price.addEventListener("input", () => {
...
+```
+
+```css hidden live-sample___date1
+input:invalid + span:after {
+  content: " ✖";
+}
+
+input:valid + span:after {
+  content: " ✓";
+}
```
Comment (hamishwillee) [inline files/en-us/learn/forms/html5_input_types/index.md:260]: This is all hidden, I presume that is the intent?

File: files/en-us/learn/forms/html5_input_types/index.md [hunk 5]
```diff
@@ -193,97 +224,146 @@ price.addEventListener("input", () => {
...
 
 ## Date and time pickers
 
-Gathering date and time values has traditionally been a nightmare for web developers. For a good user experience, it is important to provide a calendar selection UI, enabling users to select dates without necessitating context switching to a native calendar application or potentially entering them in differing formats that are hard to parse. The last minute of the previous millennium can be expressed in the following different ways, for example: 1999/12/31, 23:59 or 12/31/99T11:59PM.
+Generally, for a good user experience when gathering date and time values, it is important to provide a calendar selection UI. These enables users to select dates without needing to context switch to a native calendar application or potentially entering them in differing formats that are hard to parse. The last minute of the previous millennium can be expressed in the following different ways: `1999/12/31`, `23:59`, or `12/31/99T11:59PM`.
```
Comment (hamishwillee) [inline files/en-us/learn/forms/html5_input_types/index.md:241]:
```suggestion
Generally, for a good user experience when gathering date and time values, it is important to provide a calendar selection UI. These enable users to select dates without needing to context switch to a native calendar application or potentially entering them in differing formats that are hard to parse. The last minute of the previous millennium can be expressed in the following different ways: `1999/12/31`, `23:59`, or `12/31/99T11:59PM`.
```

**Review by hamishwillee — APPROVED** [review]
Timely changes. I've approved, but you might want to look at https://github.com/mdn/content/pull/35514/files#r1722488121

File: files/en-us/learn/forms/html5_input_types/index.md (hunk 4 above)
Comment (estelle) [reply files/en-us/learn/forms/html5_input_types/index.md:260]: It originally linked to an example with a lot of fallback JS. This hidden part was the only part needed.
